tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -111,6 +112,41 @@ func clearKernelCache() error {
 	return nil
 }
 
+func compareFileContents(t *testing.T, fileName string, filecontent string) {
+	// After write, data will be cached by kernel. So subsequent read will be
+	// served using cached data by kernel instead of calling gcsfuse.
+	// Clearing kernel cache to ensure that gcsfuse is invoked during read operation.
+	err := clearKernelCache()
+	if err != nil {
+		t.Errorf("Clear Kernel Cache: %v", err)
+	}
+
+	content, err := os.ReadFile(fileName)
+	if err != nil {
+		t.Errorf("Read: %v", err)
+	}
+
+	if got := string(content); got != filecontent {
+		t.Errorf("File content %q not match %q", got, filecontent)
```

Comment:
rename the error message to 
File content doesn't match. Expected: %q, Actual: %q